spring-core/src/main/java/org/springframework/core/MethodParameter.java

```diff
@@ -759,7 +765,47 @@ else if (ctor != null) {
 			}
 			return false;
 		}
-		
-	}
 
+		/**
+		 * Returns a return type of a method using Kotlin Reflection API.
+		 * Introduced to support suspending functions.
+		 */
+		@NonNull
```

Comment:
Not necessary, already the default via package level `@NonNullApi`.

Reply:
Fixed.